cabal-dev-scripts/src/GenUtils.hs

```diff
@@ -160,9 +161,11 @@ toConstructorName t = t
     f c   = c
 
     special :: Text -> Text
-    special "0BSD"          = "NullBSD"
-    special "389_exception" = "DS389_exception"
```

Comment:
Note we are removing ad-hoc cases (constructors are public API via `Cabal-syntax`).